src/main/kotlin/net/irisshaders/lilybot/extensions/events/LogUploading.kt

```diff
@@ -167,23 +176,24 @@ class LogUploading : Extension() {
 		}
 	}
 
-	@OptIn(InternalAPI::class)
-	private suspend fun postToHasteBin(text: String): String {
-		val client = HttpClient()
-
-		var response = client.post("https://www.toptal.com/developers/hastebin/documents") {
-			body = text
-		}.content.toByteArray().decodeToString()
-
-		if (response.contains("\"key\"")) {
-			response = "https://www.toptal.com/developers/hastebin/" + response.substring(
-				response.indexOf(":") + 2,
-				response.length - 2
-			)
-		}
+	@kotlinx.serialization.Serializable
```

Comment:
If you `import kotlinx.serialization.Serializable` then you can just annotate with `@Serializable` no need for the whole long thing. See the data classes in `DatabaseHelper.kt` if you're unclear on how this works.

Reply:
I actually cannot, due to another @Serializable tag already existing I have to differentiate between the two. This is the only method I know of to do it